.github/CHANGELOG.md

```diff
@@ -285,6 +285,9 @@
 
 <h3>Improvements</h3>
 
+* The `./pennylane/ops/qubit.py` file is broken up into a folder of eight separate files.
```

Comment:
The number of files here doesn't seem to match the other mention (there it's six).

Reply:
```suggestion
* The `./pennylane/ops/qubit.py` file is broken up into a folder of six separate files.
```

I think there was eight at some point...